src/enabled.js

```diff
@@ -1,7 +1,17 @@
-export default function getEnabledStatus(enabledStatus) {
+function getProfilerEnabledStatus(enabledStatus) {
   return (
     enabledStatus ||
     (process.env.IOPIPE_ENABLE_PROFILER === 'true' ||
       process.env.IOPIPE_ENABLE_PROFILER === true)
   );
 }
+
+function getHeapSnapshotEnabledStatus(enabledStatus) {
+  return (
+    enabledStatus ||
+    (process.env.IOPIPE_ENABLE_HEAPSNAPSHOT === 'true' ||
```

Comment:
Consider namespacing environment variables per the plugin, i.e. `process.env.IOPIPE_PROFILER_ENABLE_HEAP`